changelog.md

```diff
@@ -1,5 +1,46 @@
 # Change log
 
+## 0.9.26
+
+### Added
+
+* **New solver**: Pumpkin LCG solver (by the ConSol Lab at TU Delft) [#669](https://github.com/CPMpy/cpmpy/pull/669)
+* **New solver**: Pindakaas library for transforming pseudo Boolean to CNF [#600](https://github.com/CPMpy/cpmpy/pull/600)
+* **New global**: `Regular` [#677](https://github.com/CPMpy/cpmpy/pull/677)
+* **New tool**: Tooling for the XCSP3 format: dataset, loader, CLI, benchmarking [#597](https://github.com/CPMpy/cpmpy/pull/597)
+* Common Subexpression Elimination (CSE) with normalized expressions [#679](https://github.com/CPMpy/cpmpy/pull/679)
+* Integer to boolean direct encoding [#653](https://github.com/CPMpy/cpmpy/pull/653)
```

Comment:
"Support for intger constraints for Boolean solvers using new `int2bool` transformation"